src/popup/App.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div v-bind:class="['popup', {'popup--dark': dark}]">
+    <h2 class="popup__header">Mintbean Editor 2020-04-04 Bryce McMath</h2>
+    <select class="popup__select" name="langs" id="langs">
+      <option v-for="(opt, i) in langOpts" :key="i" :value="opt">{{opt}}</option>
+    </select>
+
+    <editor
+      class="popup__editor"
+      v-model="content"
+      @init="editorInit"
+      :lang="lang"
+      :theme="dark ? 'twilight': 'chrome' "
+      width="500"
+      height="100"
+    ></editor>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Popup',
+  components: {
+    editor: require('vue2-ace-editor'),
+  },
+  data() {
+    return {
+      content: '',
+      langOpts: ['css'],
+      lang: 'css',
+    };
+  },
+  computed: {
+    dark() {
```

Comment:
Adapting to the user's colour scheme, that's a really nice touch!

Reply:
don't see the link to the project...but this sounds great

Reply:
> don't see the link to the project...but this sounds great

There are photos and a gif in the repo. If you want to try it out I can show you how, just let me know!